dm/master/server_test.go

```diff
@@ -175,8 +175,16 @@ func TestMaster(t *testing.T) {
 		t.Fatal(err)
 	}
 	// inject *testing.T to testMaster
-	s := testSuite.(*testMaster)
-	s.testT = t
+	s, ok := testSuite.(*testMaster)
+	if ok {
+		s.testT = t
+	}
+
+	// inject *testing.T to openAPISuite
+	os, ok := ts.(*openAPISuite)
+	if ok {
+		os.testT = t
```

Comment:
This is because the unit tests that follow will need to use the 
`testing.T`

https://github.com/Ehco1996/dm/blob/7c01c9cacf60d882548e0bbcc2486d444c9bace2/dm/master/openapi_test.go#L811